sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Azure.IoT.TimeSeriesInsights.csproj

```diff
@@ -10,6 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
+  </ItemGroup>
+
+  <!-- These packages are included as part net6.0; the external references are not needed. -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
For maintainability should this be inverted so that adding future frameworks doesn't need the conditions to be touched?